docs/docs/migration-guide.mdx

```diff
@@ -28,6 +28,13 @@ this new version for inference.
 
 Please check whether your trained model still performs as expected and retrain if needed.
 
+### NLU JSON Format
+
+[NLU training data](nlu-training-data.mdx) in JSON format is deprecated and will be
+removed in Rasa Open Source 4.0.
```

Comment:
Should we also add that the command will be removed too in 4.0?

Reply:
We still support the convert command for users migration from LUIS/Watson/Dialogflow, don't we?

Reply:
I was confused as in the meeting I thought the consensus was towards removing the command, but your point makes sense.